projects/plugins/jetpack/extensions/blocks/like/like.php

```diff
@@ -110,7 +99,7 @@ function render_block( $attr, $content, $block ) {
 		$url_parts          = wp_parse_url( $url );
 		$domain             = $url_parts['host'];
 		$show_avatars_param = $show_avatars ? '' : '&amp;slim=1';
-		$src                = sprintf( 'https://widgets.wp.com/likes/?ver=%1$s#blog_id=%2$d&amp;post_id=%3$d&amp;origin=%4$s&amp;obj_id=%2$d-%3$d-%5$s%6$s&amp;block=1%7$s', rawurlencode( JETPACK__VERSION ), $blog_id, $post_id, $domain, $uniqid, $new_layout, $show_avatars_param );
+		$src                = sprintf( 'https://widgets.wp.com/likes/index.html?ver=%1$s#blog_id=%2$d&amp;post_id=%3$d&amp;origin=%4$s&amp;obj_id=%2$d-%3$d-%5$s%6$s&amp;block=1%7$s', rawurlencode( JETPACK__VERSION ), $blog_id, $post_id, $domain, $uniqid, $show_avatars_param );
```

Comment:
We are missing one argument here (i.e. we have removed `$new_layout`, but kept all arguments in the string). This results in the following PHP error:

```php
PHP Fatal error:  Uncaught ArgumentCountError: 8 arguments are required, 7 given in /srv/htdocs/wp-content/plugins/jetpack-dev/extensions/blocks/like/like.php:102
```

Reply:
Good spot, fixed!